apps/session-manager/package.json

```diff
@@ -0,0 +1,44 @@
+{
+    "name": "@pagopa/io-session-manager",
+    "version": "0.0.1",
+    "description": "API for IO session",
+    "main": "./dist/main.js",
```

Comment:
is this needed? what is the difference with `dist/server.js`

Reply:
Removed in [[#IOPID-1763] add eslint + fix based on code review](https://github.com/pagopa/io-auth-n-identity-domain/pull/1/commits/95787ff45a3ddbfeccbb0f616623810c957e883f)